jaypeasee/sorting-hat-quiz
# Feature - Custom Styling

#### What’s this PR do?
Significantly improves the page UI with dynamic styling for:
1. The `Result` component (each of the 4 results have their own colors)
2. The `Error` component to take props that dictate the message.
3. Each slide text color.
4. Resets the quiz reset functionality to redirect to `Question` 1.

#### Where should the reviewer start?
* `Result.js`
* `Error.js`
* All stylesheets

#### How should this be manually tested?
1. Run `npm start` to start the quiz.
2. Take it 4 times to get a different house result and see the colors.
3. Reset the quiz from the results page.

#### What are the relevant tickets?
* #27 
* #25 
* #24 

#### Screenshots (if appropriate):

![Screen Shot 2021-01-14 at 4 47 29 PM](https://user-images.githubusercontent.com/59029768/104662493-333b5700-5688-11eb-9d71-b2c18f24d3cd.png)

![Screen Shot 2021-01-14 at 4 48 01 PM](https://user-images.githubusercontent.com/59029768/104662528-464e2700-5688-11eb-9459-945696b97fb5.png)

![Screen Shot 2021-01-14 at 4 48 34 PM](https://user-images.githubusercontent.com/59029768/104662559-5a922400-5688-11eb-8c02-c685e46d57ab.png)

![Screen Shot 2021-01-14 at 4 48 56 PM](https://user-images.githubusercontent.com/59029768/104662580-667de600-5688-11eb-927b-c71f523d90d7.png)
